bundle/tests/scorecard/kuttl/deployment-strategy/01-remove-rolling-update.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: deployment-strategy-rc
+spec:
+  # Add fields here
+  replicas: 3
+  deployment:
+    updateStrategy:
+      rollingUpdate: null
+      type: Recreate
```

Comment:
Following this check, we should remove updateStrategy altogether and make sure Deployment no longer has the Recreate type (Deployment will default to RollingUpdate). Same for StatefulSet.